react-app-drawer-component/src/appDrawer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
```

Comment:
props is always available with `this.props` you don't have to assign it to a property.